common/src/main/scala/com/gu/recipeasy/models/CuratedRecipeDB.scala

```diff
@@ -0,0 +1,23 @@
+package com.gu.recipeasy.models
+
+//this model is different from CuratedRecipe as it holds a list of Tag ids
```

Comment:
I tend to nest the DB model inside the domain model's companion object, just so I don't need to think of a good name for it 😉  e.g. https://github.com/guardian/amigo/blob/master/app/models/Bake.scala#L16